manuscript/draft.Rmd

```diff
@@ -1,47 +1,58 @@
-Demonstrate best practices \& their contribution to better predictions
-  * Use long-term data sets such as BBS to evaluate predictive skill
-  * Note the large change in average temperatures since the 80's; comparable to 
-    the potential changed in the next 20 years 
-  * Lots of talk about how we should do it, fewer examples of doing them all together
-  * Examples of best practices
-    * Not just bioclim
-    * Dynamic predictor variables (not just site-level averages over 50 years like 
-      Worldclim)
-    * time series models (temporal autocorrelation)
-    * spatial/site-level stuff (repeated measures aren't fully independent)
-    * Properly handling uncertainty:
-      * probabilistic/error bars
-      * error propagation
-    * multiple models
-      * Test with simple baselines (e.g. random walk, future-is-like-present)
-  * Focus on richness/diversity because \[citations saying that it's important\]
-  * Publish forecasts at different timescales to evaluate performance/credibility
-Here we do that:
-  * forecasts for biodiversity in NAmerican birds with as many best practices as 
-    possible
-  * Discuss the influence of these improved methods \& of our error rates on our 
-    ability to understand:
-    * How to improve
-    * How much to trust the predictions
+Forecasting the future state of ecological systems is increasingly important for
+planning, management, and testing the performance of ecological models
+[@clark2001, @dietz2017]. One area of particular interest for forecasting is
+biodiversity since it is important for ecosystem function, central to
+conservation planning, and expected to be influenced by anthropogenic global
+change. Anticipating potential changes in biodiversity is crucial for large
+scale management and conservation, and for addressing debates regarding whether
+local scale diversity is declining and may continue to decline in response to
+anthropogenic influences [@dornelas2014].
+
+Previous efforts to predict future patterns of species richness, and diversity
+more generally, have focused primarily on building species distributions models
+to quantify the spatial relationships between the occurrence of individual
+species and the environment. Forecasts for future environmental conditions are
+then used to predict where each species will occur in the future and the
+probabilities of occurrence are summed across species to predict future species
+richness [e.g., @thomas2004]. Alternatively, models that directly relate spatial
+patterns of species richness to the environment have been developed and
+generally perform equivalently to species distribution modeling based methods
+[@algar2009, @distler2015].
+
+However, despite the emerging interest in forecasting species richness and other
+aspects of biodiversity, little is known about how effectively we can anticipate
+their dynamics. What we do know comes from a small number of hindcasting
```

Comment:
Do we want to be more explicit that evaluation of long-term forecasting is hard because of time lag before we find out how well we did?